helmfile/upstream/dex/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: 2.24.0
+appVersion: 2.28.1
 deprecated: true
 description: DEPRECATED OpenID Connect Identity (OIDC) and OAuth 2.0 Provider with
```

Comment:
I am confused. Where did you get this Chart from. The one [here](https://github.com/dexidp/helm-charts/blob/master/charts/dex/Chart.yaml) does not have the deprecation warning.